chemclipse/plugins/org.eclipse.chemclipse.chromatogram.xxd.peak.detector.supplier.firstderivative/src/org/eclipse/chemclipse/chromatogram/xxd/peak/detector/supplier/firstderivative/core/FirstDerivativePeakDetector.java

```diff
@@ -70,7 +70,7 @@ public String getName() {
 				} else {
 					configuration = globalConfiguration;
 				}
-				slopes = PeakDetectorMSD.getFirstDerivativeSlopes(new ChromatogramSelectionMSD((IChromatogramMSD)measurement), configuration.getMovingAverageWindowSize());
+				slopes = PeakDetectorMSD.getFirstDerivativeSlopes(new ChromatogramSelectionMSD((IChromatogramMSD)measurement), configuration);
```

Comment:
The PeakDetectorMSD.getFirstDerivativeSlopes should not contain the configuration object but only required parameters so it could be used outside the context.

Reply:
Resolved with 96ddd0625efcb7073a2c87dd4f8eb46bee8e2ec6.